src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -296,6 +362,216 @@ const formConfig = {
+
+                      return payload;
+                    },
+                    parseResponse: (response, file) => {
+                      return {
+                        name: file.name,
+                        confirmationCode: response.data.attributes.guid
+                      };
+                    },
+                    attachmentSchema: {
+                      'ui:title': 'Document type'
+                    },
+                    attachmentName: {
+                      'ui:title': 'Document name'
+                    }
+                  }),
+                  { 'ui:description': () => (<div>
+                    <p>File upload guidelines:</p>
+                    <ul>
+                      <li>File types you can upload: .pdf, .jpeg, .gif, .tiff, or .png</li>
+                      <li>Maximum file size: 25 MB</li>
+                    </ul>
+                    <p><em>Large files can be more difficult to upload with a slow Internet connection</em></p>
+                  </div>) })
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              disabilities: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  properties: {
+                    medicalRecords: {
+                      type: 'array',
+                      items: {
+                        type: 'object',
+                        properties: {
+                          name: {
+                            type: 'string'
+                          },
+                          size: {
+                            type: 'integer'
+                          },
+                          confirmationCode: {
+                            type: 'string'
+                          },
+                          attachmentId: {
+                            type: 'string',
+                            'enum': [
+                              '1',
+                              '2',
+                              '3',
+                              // '4',
+                              '5',
+                              '6'
+                            ],
+                            enumNames: [
+                              'Discharge',
+                              'Marriage related',
+                              'Dependent related',
+                              // 'VA preneed form',
+                              'Letter',
+                              'Other'
+                            ]
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+        // pageFive: {},
         // pageSix: {},
         // pageSeven: {},
         // pageEight: {},
```

Comment:
Looks like this PR is supposed to cover page eight (4.8). I think it'll help with merge conflicts if you overwrite `// pageEight: {}` with all your changes.

Reply:
Sounds good, changed.